ansible/roles/etcd/defaults/main.yaml

```diff
@@ -9,8 +9,13 @@ etcd_peer_port: 2380
 etcd_client_legacy_port: 4001
 etcd_peer_legacy_port: 7001
 etcd_peers_group: etcd
+
 etcd_url_scheme: http
 etcd_peer_url_scheme: http
+# Set correspondingly to true when use https values above to enable cert auth
+etcd_client_cert_auth: true
+etcd_peer_client_cert_auth: true
```

Comment:
Both ``ETCD_CLIENT_CERT_AUTH`` and ``ETCD_PEER_CLIENT_CERT_AUTH`` are false by default.

Setting both to ``true`` changes the default behaviour of deployed etcd. I would rather set it to false and leave the final decision to operator. What do you think?

[1] https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--client-cert-auth
[2] https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--peer-client-cert-auth